### PR TITLE
Update for version 2.9.5

### PR DESCRIPTION
**Pull request by bow**
Hello everyone,

As mentioned in #7, this is my PR for the update to Jinja 2.9.5. Tests are passing on [all](https://circleci.com/gh/bow/jinja2-feedstock/2) [of the CI](https://ci.appveyor.com/project/bow/jinja2-feedstock) [platforms we use](https://travis-ci.org/bow/jinja2-feedstock/builds/202063353), so I guess this is a good sign?

In any case, would anyone mind checking if everything is in order? I may have missed some things :).

Thanks in advance!

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,10 +7,10 @@ package:
 source:
   fn: Jinja2-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/J/Jinja2/Jinja2-{{ version }}.tar.gz
-  md5: edb51693fe22c53cee5403775c71a99e
+  md5: 9e55f0db6620dd99a1b366183a94270d
```
Comment (jakirkham) [inline recipe/meta.yaml]: Feel free to switch to the `sha256` from [pypi.org]( http://pypi.org/ ).

Comment (bow) [reply recipe/meta.yaml]: Good point ~ switched in 894b429.

**Review by jakirkham — APPROVED** [review]

**Comment by bow** [comment]
Hello everyone,

Just wondering if anything else needs to be changed :). Let me know if that is the case ~

**Comment by scopatz** [comment]
Thanks @bow!

**Pull request merged**